TestApp/Source/macOS/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,717 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Sorry for lack of macOS development knowledge, are these files generated by Xcode? Have you changed them?

Reply:
generated by xcode and untouched